bin/npg_qc_tag_sniff.pl

```diff
@@ -14,6 +14,11 @@
 use Carp;
 use Getopt::Long;
 use Term::ANSIColor qw(:constants);
+use LWP::Simple qw(get);
+use JSON;
+
+# this URL returns a complete set of known tags in json format
+our $LIMS_TAGS_URL = q[https://sequencescape.psd.sanger.ac.uk/api/v2/tag_groups];
```

Comment:
There is a variable in the config file `npg_qc_viewer.conf` which has the first part of the url `https://...ac.uk/` which we could use here to avoid having it hardcoded.